contrib/devtools/github-merge.py

```diff
@@ -218,6 +218,11 @@ def main():
                 print("ERROR: Merge rejected.",file=stderr)
                 exit(7)
 
+        second_sha512 = tree_sha512sum()
```

Comment:
Looks like we missed https://github.com/bitcoin/bitcoin/pull/9871 completely, resulting in this backported pr to not "move" code but instead introduce incomplete code.